src/MapWindow/GlueMapWindowItems.cpp

```diff
@@ -49,6 +49,10 @@ GlueMapWindow::ShowMapItems(const GeoPoint &location,
   const MoreData &basic = CommonInterface::Basic();
   const DerivedInfo &calculated = CommonInterface::Calculated();
 
+  bool doRefreshMapList = false;
+
+  do{
+
   bool only_landable = false;
```

Comment:
Bad indent

Reply:
Oh yes, but the correct indent would have deleted a lot of code lines and added new ones without the functional changes. I change that.